tests/test_other.py

```diff
@@ -9439,7 +9439,7 @@ def test(args):
     # Changing this option to [] should decrease code size.
     self.assertLess(changed, normal)
     # Check an absolute code size as well, with some slack.
-    self.assertLess(abs(changed - 5001), 150)
```

Comment:
Why does this regress? Seems odd this would be in the other direction than the others.

Reply:
Sorry I should have called that out.   I just because its been forever since we updated this (its not automatic via --rebase)

Reply:
Ok to land?